docker-storage-setup.1

```diff
@@ -36,9 +36,10 @@ STORAGE_DRIVER:
       overlay2 and "". Empty string tells docker-storage-setup to
       not do any storage setup.
 
-EXTRA_DOCKER_STORAGE_OPTIONS:
+EXTRA_STORAGE_OPTIONS:
       A set of extra options that should be passed to the Docker
       daemon.
+      NOTE: This option replaces EXTRA_DOCKER_STORAGE_OPTIONS which has been deprecated
```

Comment:
How about following format for options in man page.

EXTRA_DOCKER_STORAGE_OPTIONS: (DEPRECATED)
EXTRA_STORAGE_OPTIONS:
       Description of option....